src/Components/CreateImageWizardV2/validators.ts

```diff
@@ -36,3 +38,38 @@ export const isBlueprintNameValid = (blueprintName: string) =>
 export const isBlueprintDescriptionValid = (blueprintDescription: string) => {
   return blueprintDescription.length <= 250;
 };
+export const isFileSystemConfigValid = (partition: Partition[]) => {
+  const duplicate = getDuplicateMountPoints(partition);
+  return !(
+    (duplicate?.duplicates?.length !== 0 &&
+      duplicate?.duplicates?.length !== undefined) ||
+    duplicate?.root === false
+  );
+};
+export const getDuplicateMountPoints = (partition: Partition[]) => {
```

Comment:
I think we can refactor this function to make it a lot simpler. Right now it can return either undefined, or an object with a duplicates field that can be undefined, as well as a root field that doesn't seem related to the duplicate mountpoints... it is very confusing.

I think that it should return either (1) an array of partition ids that have duplicate mount points or (2) an array of duplicate mountpoints (just strings). In case there are no duplicate mount points, it would just return an empty array.

Reply:
I don't think we need it anymore do we? The root '/' mountpoint is provided by default, and you aren't allowed to add another one - you can only add mountpoints from the list:
![image](https://github.com/osbuild/image-builder-frontend/assets/95542540/fec8de3c-38cf-46e6-b8b9-481f491052e6)

Reply:
there is an error in case that there is no root 
 getDuplicateMountPoints(partitions)?.root === false && (
          <div style={{ padding: '15px 0' }}>
            <Alert
              isInline
              variant="danger"
              title="No root partition configured."
            />
we can remove it?

Reply:
I think so... I don't see how you could ever get to the state where you have two '/' partitions since '/' isn't an option the user can select.